Eval/bitvec_290__292.lean

```diff
@@ -2,5 +2,5 @@ import LeanSAT.Reflect.Tactics.BVDecide
 open BitVec
 
 theorem bitvec_290__292 :
- ∀ (Y Op1 : BitVec 64), 1#64 <<< Y * Op1 = Op1 <<< Y
+ ∀ (Y Op1 : BitVec 8), 1#64 <<< Y * Op1 = Op1 <<< Y
```

Comment:
I am surprised the `1#64` still remains in this expression? Maybe replace it with `1`? Also, why does this still typecheck?

Reply:
It doesn't but this test is marked as failure because it contains a shift /o\